server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -395,6 +398,15 @@ private GitWithAuth openGit(File workDir) throws Exception {
         }
     }
 
+    private static String generateCommitDetail(RevCommit headCommit) {
+        final PersonIdent authorIdent = headCommit.getAuthorIdent();
+        return "The commit information of " + ": " + '\n' +
```

Comment:
- It seems `information of <somethinig>` is missing.
- No need to append `\n` separately. 
```suggestion
        return "The commit information of " + ":\n" +
```

Reply:
How about this format:

```
Remote commit:
- SHA: <sha256>
- Subject: <the first line>
- Author: name <email>
- Date: ...
```

Reply:
Hi @ikhoon, as @minwoox suggest in comment, `abbrId` has been in the summary already, so we don't need it in detail, how do you think?

Reply:
Yes, I agree with your suggestion, let me update this. Thank you @trustin cc @ikhoon